ruby_basics/README.md

```diff
@@ -7,7 +7,7 @@ These exercises are designed to complement the [Ruby Basic lessons](https://www.
 2. Run the tests for an exercise file, for example `bundle exec rspec spec/numbers_exercises_spec.rb`. The first test will fail and the rest will be skipped.
 3. Open that corresponding exercise file in your text editor, in this case the `exercises/numbers_exercises.rb` file.
 4. Write the code to get the failing test to pass in the exercise file and run the tests again to verify it passes.
-5. Go into the test file and unskip the next test by removing the `x` from `xit`.
+5. Each `exercises` folder has a corresponding `spec` folder with matching test files. For instance, you can find the tests for `exercises/numbers_exercises.rb` in `spec/numbers_exercises_spec.rb`.<br>Open the matching spec file and unskip the next test by removing the `x` from `xit`.
```

Comment:
I am not a fan on using a `<br>` to break up this lengthy instruction, although I really like the idea of being very explicit. What do you think about splitting up these instructions like "5. Open the file. 6. Unskip the next text"?

Reply:
Honestly I think I agree. That was the original way I approached it, but I changed it because the step wasn't an instruction like every other bullet but just a description. 

That was my reasoning; happy to change it back. Fairly minor either way 😅